backstop.json

```diff
@@ -16,2091 +16,1202 @@
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-icon--small"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
+      "delay": 20
     },
     {
       "label": "Icons - medium",
       "url": "http://localhost:3000/section/1.3-3/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-icon--medium"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
+      "delay": 20
     },
     {
       "label": "Icons - large",
       "url": "http://localhost:3000/section/1.3-4/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-icon--large"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
+      "delay": 20
     },
     {
       "label": "Topbar",
       "url": "http://localhost:3000/section/1.6/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .l-topbar"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 100,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
-    },
+      "delay": 100},
     {
       "label": "Clearfix",
       "url": "http://localhost:3000/section/1.7/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-clearfix"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
+      "delay": 20
     },
     {
       "label": "Wordbreak",
       "url": "http://localhost:3000/section/1.8/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-break-long-words"
       ],
-      "readyEvent": "styleguideRendered",
-      "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
-      "onReadyScript": ""
+      "delay": 20
     },
     {
       "label": "Expander",
       "url": "http://localhost:3000/section/1.9/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section"
       ],
-      "readyEvent": "styleguideRendered",
       "delay": 20,
-      "misMatchThreshold" : 0.1,
-      "onBeforeScript": "",
       "onReadyScript": "clickButton"
     },
     {
       "label": "Flex",
       "url": "http://localhost:3000/section/1.10/fullscreen",
-      "hideSelectors": [],
-      "removeSelectors": [],
       "selectors": [
         ".object-section .o-flex"
```

Comment:
Can we also normalise these to all use simply `.object-section`? There are a couple of exceptions — the two modals — which we’d need to be able to override